great_expectations/experimental/datasources/pandas_datasource.py

```diff
@@ -371,7 +178,7 @@ def add_parquet_asset(
         order_by: Optional[BatchSortersDefinition] = None,
         **kwargs,  # TODO: update signature to have specific keys & types
     ) -> ParquetAsset:  # type: ignore[valid-type]
-        """Adds a parquet asset to this pandas datasource
+        """dds a Parquet DataAsst to the present "PandasDatasource" object.
```

Comment:
```suggestion
        """Adds a Parquet DataAsst to the present "PandasDatasource" object.
```

Reply:
@Kilo59 Fixed (sorry -- and thanks for catching!).